client/assets/images/README.md

```diff
@@ -0,0 +1,8 @@
+# Images
+
+Images from this folder are managed by Webpack, and can be imported like this:
+
+```javascript
+import mediaPostImage from 'assets/images/upgrades/media-post.svg';
+```
+Note this creates an immutable name with a hash suffix for the image asset: the url  will change if the source image changes.
```

Comment:
To help the reader figure out how it all works: when the image file is imported like this, webpack knows that it's used by the app and will copy it automatically to the `public/images` output folder. With an immutable hashed name.